packages/sdk/src/syncedStreamsLoop.ts

```diff
@@ -217,6 +218,9 @@ export class SyncedStreamsLoop {
                 this.log('stopSync syncState', syncState)
                 this.log('stopSync syncLoop', syncLoop)
                 this.log('stopSync syncId', syncId)
+                if (syncId) {
+                    this.abortControllers[syncId]?.forEach((x) => x.abort())
```

Comment:
I'm not familiar with react and don't understand the relation between abortController and cancelSync. But would it make sense to delete `abortController[syncId]` always after this?